appIndicator.js

```diff
@@ -404,6 +404,13 @@ class AppIndicators_IconActor extends St.Icon {
     }
 
     _createIconByName(path, callback) {
+        // We must make an early return on null case to gracefully handle null exception.
+        if (path === null || path === undefined){
+            Util.Logger.warn(`${this._indicator.id}, Impossible to read image info from path 'null'`);
```

Comment:
```suggestion
            Util.Logger.warn(`${this._indicator.id}, Impossible to read image info from 'null' path`);
```

Reply:
I believe it was done this way to be consistent with the other error message in the same function.